app/mailers/generic_mailer.rb

```diff
@@ -4,9 +4,9 @@ class GenericMailer < ApplicationMailer
   default from: %("Mailgun Sandbox" \
     <postmaster@sandbox8f696e611a6e4906a977c007f8971322.mailgun.org>)
 
-  def drop_duty(duty)
+  def drop_duty(duty, user_id)
```

Comment:
Why not just take in `users`, and instead of passing 0 you can pass in `User.all` something like that